src/Filters.js

```diff
@@ -0,0 +1,64 @@
+function Filters(props) {
+  // A function that takes the argument e which is the event object from the listener, then checks if the value
```

Comment:
I really like your use of comments, is this to help remind yourself and cement the learning on certain lines of code?

Reply:
Yeah exactly, it helps with remembering my decisions and also semi-rubberducking myself with the longer winded logic.